src/general/EditorBase.cs

```diff
@@ -136,6 +138,21 @@ public GameProperties CurrentGame
 
     public bool NodeReferencesResolved { get; private set; }
 
+    [JsonIgnore]
```

Comment:
The editor remembers the symmetry setting (unless I'm misremembering) so why shouldn't the editor also remember the selected light level (I assume there's a button to select one)?